test/provision_test.py

```diff
@@ -88,21 +80,15 @@ def test_with_consecutive_repeated_wells(self):
         assert expected_instruction_as_json == actual_instruction_as_json
 
     def test_with_repeated_wells_but_discontinious(self):
```

Comment:
nit: correct typo `discontinious` to `discontinuous` (since we're changing this method I feel it's within scope)

Reply:
Fixed